### PR TITLE
Update LoadErrorMessage components

### PR DESCRIPTION
**Pull request by mattcorner**
## Changes

Breaking:  
- Replaced `showImg` property with `image`. Rather than a `{showImg: boolean}` these are now set using `{image: "none" | "virto-thinking" | "virto-shrugging" | "virto-head-scratching"}`
- Updated `supportUrl` prop name to `contactUrl` to align with the `contactTeam` property.

New: 
- Adds new image types for the card. These include virto-shrugging, virto-head-scratching and virto-thinking. These have also been updated to be a reasonable size rather that high-resolution. 
- Updated `contactTeam` to support a "none" value. This can be used to not show the contact message.
- Fixed contact link not rendering correctly e.g. correct pointer. 
- Updated to latest storybook.
- Exposed default props for each use case we currently have. Included is default404Props, default401Props and defaultErrorProps
    ```
    <LoadErrorMessage {...LoadErrorMessage.default404Props} />
    ```

Storybook:
- Update the stories for this component to match the Figma provided by @Sowbhagya-ipg https://www.figma.com/file/5qkWskMGpXY38REVIDcJnW/Landing-Page?type=design&node-id=2921%3A4413&mode=dev&t=ZqF63hVNXKpQ64Yr-1

## Dependencies

To be used in https://github.com/IPG-Automotive-UK/virto/pull/552

I've included some breaking changes. This is currently only used in an unreleased staging branch and so now is the right time to improve the API.

## UI/UX

image
image

image
image

## Author checklist before assigning a reviewer

- [x] Reviewed my own code-diff.
- [x] PR assigned to me or an appropriate delegate.
- [x] Relevant labels added to the PR.
- [x] Appropriate tests have been added.
- [x] Lint and test workflows pass.

### CODE REVIEW

**Comment by Sowbhagya-ipg** [comment]
Looks Good! @mattcorner

File: src/LoadErrorMessage/LoadErrorMessage.stories.tsx [hunk 1]
```diff
@@ -19,88 +23,28 @@ const Template: StoryFn<LoadErrorMessageProps> = args => {
...
-    message:
-      "Seems like there is an issue fetching data. Please try again later.",
-    showImg: true,
-    supportUrl: "https://ipg-automotive.com/en/support/support-request/",
-    title: "Failed to fetch data"
-  },
-  render: Template
-};
+// Story type
+type Story = StoryObj<typeof LoadErrorMessage>;
```
Comment (Skoob1905) [inline src/LoadErrorMessage/LoadErrorMessage.stories.tsx:27]: Should this type belong in the types file?

**Review by Skoob1905 — COMMENTED** [review]
Code looks pretty good and storybook runs fine. Just one question about the types but other than that good to go!

**Comment by dmbarry86** [comment]
I love the new VIRTO images @Sowbhagya-ipg

File: src/LoadErrorMessage/LoadErrorMessage.tsx [hunk 2]
```diff
@@ -107,31 +122,47 @@ const LoadErrorMessage = ({
...
+  image: "virto-shrugging",
+  message:
+    "We couldn't find the page you requested. It either does not exist or you may not have access to it.",
+  title: "Page not found!"
+} as const;
+
+LoadErrorMessage.defaultErrorProps = {
+  actionButtonText: "Refresh",
+  contactTeam: "Support",
+  contactUrl: "https://support.virto.com",
```
Comment (dmbarry86) [inline src/LoadErrorMessage/LoadErrorMessage.tsx:160]: Would have been better to default this to the actual URL we need https://www.ipg-automotive.com/en/support/support-request/. Now we need to remember to set it in every app we use where we pretty much always want this same correct URL.

Comment (dmbarry86) [reply src/LoadErrorMessage/LoadErrorMessage.tsx:160]: https://github.com/IPG-Automotive-UK/virto/pull/552#issuecomment-1969102717

**Pull request merged**